### PR TITLE
feat(LoginPane): enable unstable thirdPartyAuth prop

### PR DESCRIPTION
**Pull request by cdaringe**
# problem statement

- some apps use libs that bring their own auth widgets

# solution

- support that use case

add `thirdPartyAuth` and friends to the `<LoginPane />` component

### CODE REVIEW

File: src/components/unstable/Layout/Layout.examples.md [hunk 1]
```diff
@@ -1,7 +1,11 @@
 ### Empty layout:
 
 ```js
-<Layout className='hide-in-test' logo={<h1 style={{color: 'white'}}>LOGO</h1>} />
+<Layout
+  className='hide-in-test'
+  logo={<h1 style={{color: 'white'}}>LOGO</h1>}
+  navItems={[]}
+/>
```
Comment (cdaringe) [inline src/components/unstable/Layout/Layout.examples.md]: this was generating warnings in the styleguide. just added the missing `navItems`

**Comment by wa11-e** [comment]
Woohoo! new surge deployment available for viewing! :tada: [surgereviewfbnrikpqwnahcaz.surge.sh](http://surgereviewfbnrikpqwnahcaz.surge.sh)

**Comment by cdaringe** [comment]
eager merging, unstable widget

**Pull request merged**